packages/netlify/README.md

```diff
@@ -49,81 +48,44 @@ If you prefer to install the adapter manually instead, complete the following tw
    ```diff lang="js"
      // astro.config.mjs
      import { defineConfig } from 'astro/config';
-   + import netlify from '@astrojs/netlify/functions';
+   + import netlify from '@astrojs/netlify';
 
      export default defineConfig({
    +   output: 'server',
    +   adapter: netlify(),
      });
    ```
 
-### Run middleware in Edge Functions
-
-When deploying to Netlify Functions, you can choose to use an Edge Function to run your Astro middleware.
-
-To enable this, set the `edgeMiddleware` config option to `true`:
-
-```diff lang="js"
-  // astro.config.mjs
-  import { defineConfig } from 'astro/config';
-  import netlify from '@astrojs/netlify/functions';
-
-  export default defineConfig({
-    output: 'server',
-    adapter: netlify({
-+     edgeMiddleware: true,
-    }),
-  });
-```
-
-#### Pass edge context to your site
+#### Accessing edge context from your site
 
 Netlify Edge Functions provide a [context object](https://docs.netlify.com/edge-functions/api/#netlify-specific-context-object) including metadata about the request, such as a user’s IP, geolocation data, and cookies.
 
-To expose values from this context to your site, create a `netlify-edge-middleware.ts` (or `.js`) file in your project’s [source directory](https://docs.astro.build/en/reference/configuration-reference/#srcdir). This file must export a function that returns the data to add to [Astro’s `locals` object](https://docs.astro.build/en/guides/middleware/#locals), which is available in middleware and Astro routes.
+This can be accessed via the `Astro.locals.netlify.context` object:
 
-In this example, `visitorCountry` and `hasEdgeMiddleware` would both be added to Astro’s `locals` object:
-
-```ts
-// src/netlify-edge-middleware.ts
-import type { Context } from 'https://edge.netlify.com';
-
-export default function ({ request, context }: { request: Request; context: Context }) {
-  // Return serializable data to add to Astro.locals
-  return {
-    visitorCountry: context.geo.country.name,
-    hasEdgeMiddleware: true,
-  };
-}
+```astro
+---
+const { geo: { city } } = Astro.locals.netlify.context
```

Comment:
The reason I suggested changing it is because the syntax is more confusing. For the other question, what el;se is `Astro.locals` used for? If `context` is the only other thing we're adding, I'd say just make it `context` at the top level

Reply:
It's what Astro uses to make transport data from the middleware into the SSR rendering: https://docs.astro.build/en/reference/api-reference/#astrolocals
So all other properties on there will be user-configured. `Astro.locals.context` is probably the best, yeah

Reply:
one reason that `context` might not be ideal is that Astro already has a `context` parameter they're passing into middleware. Accessing Netlify's context would look like this then:
```ts
import { defineMiddleware } from "astro:middleware";

export const onRequest = defineMiddleware(async (context, next) => {
  context.locals.populatedFromMiddleware = "I was populated from a middleware!";

  context.locals.context // <-- might be confusing
  const resp = (await next()) as Response;
  resp.headers.set("x-astro-middleware", "true");
  return resp;
});
```

this feels a little confusing? maybe we should put the `netlify` name somewhere.